### PR TITLE
Resources for Lead DE

### PR DESCRIPTION
**Pull request by kalebsofer**
Includes:
- Resources for Lead, these need reviewing by Senior/Lead eng
- Moved most ML resources from Senior --> Lead

Still need resources for the final objective ("Compares and evaluates data management solutions"), which requires general knowledge. 

Let's discuss before merging.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "cSpell.words": [
```
Comment (tf75) [inline .vscode/settings.json:2]: probably don't need this setting file unless there is an explicit reason it is in here?

File: README.md [hunk 2]
```diff
@@ -4,6 +4,8 @@ Please check out below the learning objectives for data engineer levels!
 
 - :sparkle: [Data Engineer Learning Objectives](learning_objectives/Data_Engineer.md)
 - :mechanical_arm: [Senior Data Engineer Learning Objectives](learning_objectives/Senior_Data_Engineer.md)
+- :crystal_ball: [Lead Data Engineer Learning Objectives](learning_objectives/Lead_Data_Engineer.md)
```
Comment (tf75) [inline README.md:7]: crystal_ball very good choice!

File: learning_objectives/Lead_Data_Engineer.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+# Data Engineer Learning Path
+
+This document aims to provide learning resources to help in training for a Lead Data Engineer. This list is not exhaustive and is simply to help learning some of the core concepts we have around data engineering for that level. We have given a variety of resources from articles to online courses to help with progressing towards completing these learning objectives. We have also put at the end optional certifications you can pursue to concrete your knowledge. Any comments, feedback or reports of missing/broken links please slack the cop-data channel. 
+
+## Develops optimal collection and data strategies, applying different capture techniques, determining types of data, and using IDP to optimise processes, e.g. streaming or batch.
+
+[Designing Data Intensive Applications (book)](https://www.amazon.com/gp/product/1449373321/ref=as_li_tl?ie=UTF8&camp=1789&creative=9325&creativeASIN=1449373321&linkCode=as2&tag=snirsuggestin-20&linkId=d270da1b4c8d354124107125921953d3)
+
+[Designing distributed systems (book)](https://www.udemy.com/course/the-complete-sql-bootcamp/)
```
Comment (tf75) [inline learning_objectives/Lead_Data_Engineer.md:9]: the link for this and the title don't seem to match

Comment (kalebsofer) [reply learning_objectives/Lead_Data_Engineer.md]: updated

**Comment by kalebsofer** [comment]
Updates look good to me

**Pull request merged**